packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -28,7 +29,8 @@ import {
   CORRELATION_ID,
   SIP_URI,
   _LEFT_,
-  _ID_
+  _ID_,
```

Comment:
Why do some of these constants have underscores and others don't?

Reply:
The intention was to indicate its internal @chrisadubois